support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.tsx

```diff
@@ -39,13 +40,13 @@ function SubscriptionsLandingPage({
 			NZDCountries,
 			International,
 		],
-		isNewProduct,
+		isNewProduct: hideDigiSub,
```

Comment:
Would it be better to update the prop name `isNewProduct` to `hideDigiSub`?

Reply:
Yes, I think I wanted to avoid rooting around in too many files but it'd be better to just go ahead and change the prop name to avoid future confusion.